docs/introduction.rst

```diff
@@ -83,6 +75,38 @@ container images and virtual machine (VM) disk images that often harbor
 comprehensive software stacks from an operating system with its kernel through
 system and application packages to original and custom applications.
 
-.. Some of this documentation is borrowed from the metaflow documentation and is also under
-   Apache-2.0
+Dependencies and Internal Tools
+-------------------------------
+
+ScanCode.io is essentially a `Django <https://www.djangoproject.com/>`_-based
+application wrapper around the
+`ScanCode Toolkit <https://github.com/nexB/scancode-toolkit>`_ scanning engine.
+
+The **Django framework** is leveraged for many aspects of ScanCode.io:
+
+- Database management
+- :ref:`user_interface`
+- :ref:`rest_api`
+- :ref:`command_line_interface`
+
+.. note::
+    Multiple applications from the Django eco-system are also included,
+    see the `setup.cfg <https://github.com/nexB/scancode.io/blob/main/setup.cfg>`_ file
+    for an exhaustive list of dependencies.
+
+The second essential part of ScanCode.io is the **ScanCode Toolkit**, used for the
+extraction of archives and the scanning engine.
+
+The nexB `container-inspector <https://github.com/nexB/container-inspector>`_ library
+is also a key component of ScanCode.io as the tool used for the analysis of Docker
+images, containers, root filesystems and virtual machine images.
+
+
+.. note::
+    Releases of ScanCode.io usually follow the releases of ScanCode Toolkit to ensure
+    the latest improvements of the scanning engines are available as soon as ready.
+
```

Comment:
As a common practice, ScanCode.io releases usually follow ScanCode Toolkit releases to ensure
    the latest improvements of the scanning engines are included in the latest release of ScanCode.io.